features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -81,9 +81,9 @@
 #define DEFAULT_ACCEPTMBOX_SIZE     8
 
 #ifdef LWIP_DEBUG
-#define TCPIP_THREAD_STACKSIZE      1024*2
+#define TCPIP_THREAD_STACKSIZE      1200*2
```

Comment:
This is firmly in the area of black magic. I guess it could be `#ifndef` allowing the target to override.

Or maybe better, "#if TCPIP_THREAD_STACKSIZE < 1024", setting a minimum, allowing the target to be set higher?

Wouldn't want to be dragged back down by a target.